app/src/main/java/com/google/android/soundchecker/dsd/DsfReader.kt

```diff
@@ -147,9 +148,15 @@ class DsfReader(inputStream: InputStream) {
     private fun readNextBlock(channel: Int): Boolean {
         Arrays.fill(mData[channel], 0, mData[channel].size - 1, 0x0.toByte())
```

Comment:
Is 0x00 meant to be silence? Or is 0xAA silence?
Maybe if we pad partial buffers with DSD silence then we might not hear noise.

Reply:
I think should be 0xa.